src/components/internals/create-select-styles.js

```diff
@@ -72,7 +72,6 @@ const indicatorSeparatorStyles = () => base => ({
 const dropdownIndicatorStyles = () => base => ({
   ...base,
   color: vars['--token-font-color-default'],
-  marginRight: vars['--spacing-8'],
```

Comment:
There wasn't really a point in defining `marginRight` when we overwrite it with 0 in the next line.